strawberryfields/backends/bosonicbackend/backend.py

```diff
@@ -198,7 +207,7 @@ def init_circuit(self, prog):
                 for reg in labels:
                     # All the possible preparations should go in this loop
                     if isinstance(cmd.op, Bosonic):
-                        weights, means, covs = [pars[i].tolist() for i in range(3)]
+                        weights, means, covs = [pars[i] for i in range(3)]
```

Comment:
The other preparations return weights, means and covs as arrays. I think `.tolist()` was a legacy from when the other preparations were returning these as lists. But since the others return as arrays, this should too.